dev/conductor/core/lib/src/start.dart

```diff
@@ -438,15 +440,20 @@ class StartContext extends Context {
     Version requestedVersion,
     FrameworkRepository framework,
   ) async {
-    if (incrementLetter != 'm') {
-      // in this case, there must have been a previous tagged release, so skip
-      // tagging the branch point
-      return requestedVersion;
-    }
     final String branchPoint = await framework.branchPoint(
       candidateBranch,
       FrameworkRepository.defaultBranch,
     );
+    if (await framework.isCommitTagged(branchPoint)) {
+      // The branch point is tagged, no work to be done
+      return requestedVersion;
+    }
+    assert(
+      requestedVersion.n == 0,
```

Comment:
Will this only apply to betas?

Reply:
Yeah, hopefully we don't get behind on creating the new beta branches anymore to not have to use force anymore.

Reply:
In thinking more about this, I think rather than an assert and fail, I will just print a warning to STDERR that we're skipping this and exit the function.